gap/PolygonalComplexes/modification.gd

```diff
@@ -1158,6 +1158,35 @@ DeclareOperation("SplitAllVertices", [IsPolygonalComplex]);
 DeclareOperation( "TetrahedralExtension", [IsSimplicialSurface, IsPosInt] );
 #! @EndGroup
 
+#! @BeginGroup TetrahedralReduction
+#! @Description
+#! Given a simplicial surface and a vertex of face degree 3 a new surface can be 
+#! constructed by removing the faces incident to <K>vertex</K> in <K>surface</K>.
+#! and attaching a new face to the resulting boundary edges.
+#! <Alt Only="TikZ">
+#!      \input{Image_TetraRed.tex}
+#! </Alt>
+#! So, this modification results in decreasing the number of vertices by 1, the 
+#! number of edges by 3 and the number of faces by 2. As an Example consider
+#! the double tetrahedron.
+#! <Alt Only="TikZ">
+#!      \input{Image_DoubleTetrahedron.tex}
+#! </Alt>
+#! @BeginExampleSession
+#! gap> vof:=[[1,2,4],[2,3,4],[1,3,4],[1,2,5],[1,3,5],[2,3,5]];;
+#! gap> s:=SimplicialSurfaceByVerticesInFaces(vof);
```

Comment:
Renaming s to doubleTetra or something like that would be better for the reader.